lib/webmachine/translation.rb

```diff
@@ -13,7 +13,8 @@ module Translation
     #   variables to interpolate.
     # @return [String] the interpolated string
     def t(key, options={})
-      ::I18n.t(key, options.merge(:scope => :webmachine))
+      ::I18n.translate
+      ::I18n.t(key, **options.merge(:scope => :webmachine))
     end
```

Comment:
I'm confused by the extra line here. What does the separate call to `::I18n.translate` to?

Reply:
Ugh, my apologies, it's a scalpel left in the patient. `t` is an alias to `translate` so I ended up putting a new line there just so I could CMD-B to the real definition to look at the docs... will remove it